### ISSUE TITLE
[Feature request] Modern look

### ISSUE THREAD

**Issue by Qik000** [AnySoftKeyboard/AnySoftKeyboard#2276]
Anysoft is just plain ugly and cluttered. Most of the themes look like theyre from 2005 or 2010. And no theme even simillar to an AOSP (looks the same a Gboard) is available.

Anyaoft should look like AOSP from the getgo amd have the same amount of buttons, the current state is too clutered.

**Comment by Invisigon** [comment]
I like the simple layout with the exception of the space bar graphic. I agree we need some modern themes.  We could also use way to theme AnySoftKeyboard's background and popup color etc.

**Comment by kurdikeyboard** [comment]
Yeah you right that's why i created some new Themes to my personal keyboard :) 

here are some screeshots:

<a href="https://postimages.org/" target="_blank"><img src="https://i.postimg.cc/8P8bVW8G/IMG-20200605-013317.jpg" alt="IMG-20200605-013317"/></a>

<a href="https://postimg.cc/2qZvhTVn" target="_blank"><img src="https://i.postimg.cc/xdsRr4Qp/IMG-20200605-013303.jpg" alt="IMG-20200605-013303"/></a>

<a href="https://postimages.org/" target="_blank"><img src="https://i.postimg.cc/CLjsM1Yq/IMG-20200605-013710.jpg" alt="IMG-20200605-013710"/></a>

<a href="https://postimages.org/" target="_blank"><img src="https://i.postimg.cc/dQZ9X6xn/IMG-20200605-013522.jpg" alt="IMG-20200605-013522"/></a>

<a href='https://postimages.org/' target='_blank'><img src='https://i.postimg.cc/WzZ6kpkx/IMG-20200605-013438.jpg' border='0' alt='IMG-20200605-013438'/></a>

<a href="https://postimages.org/" target="_blank"><img src="https://i.postimg.cc/SKQrvpBz/IMG-20200605-013404.jpg" alt="IMG-20200605-013404"/></a>

<a href="https://postimages.org/" target="_blank"><img src="https://i.postimg.cc/xCcP93Tf/IMG-20200605-013918.jpg" alt="IMG-20200605-013918"/></a>

<a href="https://postimages.org/" target="_blank"><img src="https://i.postimg.cc/fRC7XrVW/IMG-20200605-013618.jpg" alt="IMG-20200605-013618"/></a>

<a href="https://postimages.org/" target="_blank"><img src="https://i.postimg.cc/vBktTxcW/IMG-20200605-013545.jpg" alt="IMG-20200605-013545"/></a>

<a href="https://postimages.org/" target="_blank"><img src="https://i.postimg.cc/htcbXBP3/IMG-20200605-014036.jpg" alt="IMG-20200605-014036"/></a>

**Comment by Invisigon** [comment]
> Yeah you right that's why i created some new Themes to my personal keyboard :)
> 
> here are some screeshots:
> 
> [![IMG-20200605-013317](https://camo.githubusercontent.com/310f772d9727cf1b5ffb97e525700217793eae70/68747470733a2f2f692e706f7374696d672e63632f38503862565738472f494d472d32303230303630352d3031333331372e6a7067)](https://postimages.org/)
> 
> [![IMG-20200605-013303](https://camo.githubusercontent.com/fa6f7ced41941b92a64280a551bd66cc1823e08e/68747470733a2f2f692e706f7374696d672e63632f78647352723451702f494d472d32303230303630352d3031333330332e6a7067)](https://postimg.cc/2qZvhTVn)
> 
> [![IMG-20200605-013710](https://camo.githubusercontent.com/9663df8670e5283b6d98ec8acdffbe8a35688130/68747470733a2f2f692e706f7374696d672e63632f434c6a734d3159712f494d472d32303230303630352d3031333731302e6a7067)](https://postimages.org/)
> 
> [![IMG-20200605-013522](https://camo.githubusercontent.com/d184cbd307b8fefde7b4c2b1356c856c061e67ea/68747470733a2f2f692e706f7374696d672e63632f64515a395836786e2f494d472d32303230303630352d3031333532322e6a7067)](https://postimages.org/)
> 
> [![IMG-20200605-013438](https://camo.githubusercontent.com/efcbc4075215405636c4d855dddfcb7c41bacc8d/68747470733a2f2f692e706f7374696d672e63632f577a5a366b706b782f494d472d32303230303630352d3031333433382e6a7067)](https://postimages.org/)
> 
> [![IMG-20200605-013404](https://camo.githubusercontent.com/f043104aa9873521c324986449ab748280c8648c/68747470733a2f2f692e706f7374696d672e63632f534b51727670427a2f494d472d32303230303630352d3031333430342e6a7067)](https://postimages.org/)
> 
> [![IMG-20200605-013918](https://camo.githubusercontent.com/7ecf9a5c6e3a2468ba1b53e3d24993c56f059681/68747470733a2f2f692e706f7374696d672e63632f78436350393354662f494d472d32303230303630352d3031333931382e6a7067)](https://postimages.org/)
> 
> [![IMG-20200605-013618](https://camo.githubusercontent.com/7fc3b30df56d36f9db3d6e2b0982edf88759bd9e/68747470733a2f2f692e706f7374696d672e63632f66524337587256572f494d472d32303230303630352d3031333631382e6a7067)](https://postimages.org/)
> 
> [![IMG-20200605-013545](https://camo.githubusercontent.com/527666490fc0ae846e78478a1e3468afb6764165/68747470733a2f2f692e706f7374696d672e63632f76426b74547863572f494d472d32303230303630352d3031333534352e6a7067)](https://postimages.org/)
> 
> [![IMG-20200605-014036](https://camo.githubusercontent.com/dd676ae0260d6002a393fd9172adab5f3438e30f/68747470733a2f2f692e706f7374696d672e63632f68746362584250332f494d472d32303230303630352d3031343033362e6a7067)](https://postimages.org/)

Is that your own version of AnySoftKeyboard? The themes looks cool!

**Comment by Invisigon** [comment]
> Yeah you right that's why i created some new Themes to my personal keyboard :)
Is that your own version of AnySoftKeyboard? The themes look cool!

**Comment by teynav** [comment]
I would agree to the clutter part its hinders the typing experience  I literally hate the ʾ key beside L and can't help  to remove it. -_-

**Comment by kurdikeyboard** [comment]
> > Yeah you right that's why i created some new Themes to my personal keyboard :)
> 
> Is that your own version of AnySoftKeyboard? The themes look cool!

Yes it's mine that build from anysoftkeyboard source code :)

**Comment by Invisigon** [comment]
> > > Yeah you right that's why i created some new Themes to my personal keyboard :)
> > 
> > 
> > Is that your own version of AnySoftKeyboard? The themes look cool!
> 
> Yes it's mine that build from anysoftkeyboard source code :)

That is so cool. Wish I knew how to complie my own version of AnySoftKeboard. I would have made an AMOLED theme with the letters and icons being white instead of muted white or gray.

**Comment by kurdikeyboard** [comment]
> > > > Yeah you right that's why i created some new Themes to my personal keyboard :)
> > > 
> > > 
> > > Is that your own version of AnySoftKeyboard? The themes look cool!
> > 
> > 
> > Yes it's mine that build from anysoftkeyboard source code :)
> 
> That is so cool. Wish I knew how to complie my own version of AnySoftKeboard. I would have made an AMOLED theme with the letters and icons being white instead of muted white or gray.

The latest source code is not build in Android Studio I'm used the old stable version 1.10-r3 but it have an isuses when you work with clipboard after copying some things the app crashes and freeze the device... if want ant help from me to making Themes contact me gorran90@gmail.com making Themes to this keyboard is so easy :)
